mobile_config/src/gateway_info.rs

```diff
@@ -2,15 +2,99 @@ use chrono::{DateTime, TimeZone, Utc};
 use futures::stream::BoxStream;
 use helium_crypto::PublicKeyBinary;
 use helium_proto::services::mobile_config::{
-    DeviceType as DeviceTypeProto, GatewayInfo as GatewayInfoProto,
-    GatewayMetadata as GatewayMetadataProto,
+    gateway_metadata::DeploymentInfo as DeploymentInfoProto,
+    CbrsDeploymentInfo as CbrsDeploymentInfoProto,
+    CbrsRadioDeploymentInfo as CbrsRadioDeploymentInfoProto, DeviceType as DeviceTypeProto,
+    GatewayInfo as GatewayInfoProto, GatewayMetadata as GatewayMetadataProto,
+    WifiDeploymentInfo as WifiDeploymentInfoProto,
 };
+use serde::Deserialize;
 
 pub type GatewayInfoStream = BoxStream<'static, GatewayInfo>;
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct WifiDeploymentInfo {
+    /// Antenna ID
+    pub antenna: u32,
+    /// The height of the hotspot above ground level in whole meters
+    pub elevation: u32,
+    pub azimuth: u32,
+    #[serde(rename = "mechanicalDownTilt")]
+    pub mechanical_down_tilt: u32,
+    #[serde(rename = "electricalDownTilt")]
+    pub electrical_down_tilt: u32,
+}
+impl From<WifiDeploymentInfoProto> for WifiDeploymentInfo {
+    fn from(v: WifiDeploymentInfoProto) -> Self {
+        Self {
+            antenna: v.antenna,
+            elevation: v.elevation,
+            azimuth: v.azimuth,
+            mechanical_down_tilt: v.mechanical_down_tilt,
+            electrical_down_tilt: v.electrical_down_tilt,
+        }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct CbrsDeploymentInfo {
+    pub cbrs_radios_deployment_info: Vec<RadioDeploymentInfo>,
+}
+
+impl From<CbrsDeploymentInfoProto> for CbrsDeploymentInfo {
+    fn from(v: CbrsDeploymentInfoProto) -> Self {
+        Self {
+            cbrs_radios_deployment_info: v
+                .cbrs_radios_deployment_info
+                .into_iter()
+                .map(|v| v.into())
+                .collect(),
+        }
+    }
+}
+
+impl From<CbrsRadioDeploymentInfoProto> for RadioDeploymentInfo {
+    fn from(v: CbrsRadioDeploymentInfoProto) -> Self {
+        Self {
+            radio_id: v.radio_id,
+            elevation: v.elevation,
+        }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct RadioDeploymentInfo {
```

Comment:
should probably also prepend this with `CbrsRadioDeploymentInfo` to avoid confusion

Reply:
Fixed